### PR TITLE
feat: Show total sales of Event

### LINKED ISSUE #184: Show total sales of the Event
Show the total sale of the Event in the Event Detail Screen

test

### PR DESCRIPTION
**Pull request by iamareebjamal**
Fixes #184 

- Add total sales visual in Event Detail
- Add the corresponding test

device-2017-06-20-173258

### CODE REVIEW

**Comment by iamareebjamal** [comment]
@niranjan94 Is there any field in API to get the currency of tickets. We have to show it in dollars currently just because of generality where on the site, it is actually in INR

**Comment by iamareebjamal** [comment]
@mejariamol @arp95 @Shailesh351 @rohanagarwal94 Please review

**Comment by niranjan94** [comment]
@iamareebjamal part of the event API. `payment-currency`

**Comment by niranjan94** [comment]
@iamareebjamal `payment-currency` will be a three letter ISO 4217 currency code. (Eg. `USD`, `INR`, `GBP`)

**Comment by mejariamol** [comment]
@iamareebjamal the circular ring around gives false feeling of some graphical display. :-)

**Comment by iamareebjamal** [comment]
@mejariamol I did not understand. False feeling of some graphical display. You mean as if it is a circular progressbar?

**Comment by mejariamol** [comment]
@iamareebjamal yes

**Comment by iamareebjamal** [comment]
I thought this gives a nice touch on otherwise bland landcape. Let's see what others think

**Comment by iamareebjamal** [comment]
device-2017-06-20-185402
 How about this?

**Comment by mejariamol** [comment]
@iamareebjamal 
- please move it to the start (Left)
- little title saying `Sales` or something at top will do good.

**Comment by iamareebjamal** [comment]
The graphic won't look good on left. The title can be centralised as well

**Comment by iamareebjamal** [comment]
device-2017-06-20-185402

3rd UI

**Comment by iamareebjamal** [comment]
@arp95 @Shailesh351 @rohanagarwal94 Guys, please review which one is better

**Comment by arp95** [comment]
i find the 2nd one better.

**Comment by Shailesh351** [comment]
Yes, 2nd one is better also add "Total sales" lable in it. because it may confuse user what is it.

**Comment by iamareebjamal** [comment]
device-2017-06-20-210944

**Comment by mejariamol** [comment]
Looking good to me :+1:

**Comment by rohanagarwal94** [comment]
Yes, the 2nd one for me too.

File: app/src/main/java/org/fossasia/openevent/app/common/binding/BindingAdapters.java [hunk 1]
```diff
@@ -17,6 +18,11 @@
 
 public class BindingAdapters {
 
+    @BindingConversion
+    public static String longToStr(Long value) {
+        return value != null ? String.valueOf(value) : "";
```
Comment (mejariamol) [inline app/src/main/java/org/fossasia/openevent/app/common/binding/BindingAdapters.java]: when a long value would be null?

Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/openevent/app/common/binding/BindingAdapters.java]: Suppose a model does not include a primitive long, but the `Long` class, and it is missing from the API response. Or just if someone wanted to make the app crash and created a null long and bind it in the layout

**Pull request merged**